sources/packages/backend/workflow/test/2023-2024/parttime-assessment/costs/parttime-assessment-2023-2024-costs-total-academic-expenses.e2e-spec.ts

```diff
@@ -0,0 +1,34 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import { OfferingDeliveryOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-total-academic-expenses.`, () => {
+  it(
+    "Should determine calculatedDataTotalAcademicExpenses when offering weeks is 30, " +
```

Comment:
Maybe we can use different test data / flows in different program years to have more diversity of the tests.